src/pages/PoolDetails/PoolDetailsHeader.tsx

```diff
@@ -0,0 +1,208 @@
+import { Trans } from '@lingui/macro'
+import { ChainId, Currency } from '@uniswap/sdk-core'
+import blankTokenUrl from 'assets/svg/blank_token.svg'
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { getChainInfo } from 'constants/chainInfo'
+import { chainIdToBackendName } from 'graphql/data/util'
+import { useCurrency } from 'hooks/Tokens'
+import useTokenLogoSource from 'hooks/useAssetLogoSource'
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+import { ClickableStyle, ThemedText } from 'theme'
+import { shortenAddress } from 'utils'
+
+import { ReversedArrowsIcon } from './icons'
+
+const HeaderColumn = styled(Column)`
+  gap: 36px;
+`
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  ${ClickableStyle}
+`
+
+const FeeTier = styled(ThemedText.LabelMicro)`
+  background: ${({ theme }) => theme.surface2};
+  padding: 2px 6px;
+  border-radius: 4px;
+`
+
+const ToggleReverseArrows = styled(ReversedArrowsIcon)`
+  ${ClickableStyle}
+`
+
+interface Token {
+  id: string
+  symbol: string
+}
+
+interface PoolDetailsHeaderProps {
+  chainId?: number
+  poolAddress?: string
+  token0?: Token
+  token1?: Token
+  feeTier?: number
+  toggleReversed: React.DispatchWithoutAction
+}
+
+export function PoolDetailsHeader({
+  chainId,
+  poolAddress,
+  token0,
+  token1,
+  feeTier,
+  toggleReversed,
+}: PoolDetailsHeaderProps) {
+  const currencies = [useCurrency(token0?.id, chainId) ?? undefined, useCurrency(token1?.id, chainId) ?? undefined]
+  const chainName = chainIdToBackendName(chainId)
+  const origin = `/tokens/${chainName}`
+  return (
+    <HeaderColumn>
+      <Row>
+        <StyledLink to={origin}>
+          <ThemedText.BodySecondary>
+            <Trans>Explore</Trans>
+          </ThemedText.BodySecondary>
+        </StyledLink>
+        <ThemedText.BodySecondary>&nbsp;{'>'}&nbsp;</ThemedText.BodySecondary>
+        {/* TODO: When Explore Pool table is added, link directly back to it */}
+        <StyledLink to={origin}>
+          <ThemedText.BodySecondary>
+            <Trans>Pool</Trans>
+          </ThemedText.BodySecondary>
+        </StyledLink>
+        <ThemedText.BodySecondary>&nbsp;{'>'}&nbsp;</ThemedText.BodySecondary>
+        <ThemedText.BodyPrimary>
+          {token0?.symbol} / {token1?.symbol} ({shortenAddress(poolAddress)})
+        </ThemedText.BodyPrimary>
+      </Row>
+      <Row gap="18px">
+        <Row gap="8px" width="max-content">
+          {chainId && (
+            <DoubleCurrencyAndChainLogo data-testid="double-token-logo" chainId={chainId} currencies={currencies} />
+          )}
+          <ThemedText.HeadlineSmall>
+            {token0?.symbol} / {token1?.symbol}
+          </ThemedText.HeadlineSmall>
+        </Row>
+        {feeTier && <FeeTier>{feeTier / 10000}%</FeeTier>}
```

Comment:
```suggestion
        {Boolean(feeTier) && <FeeTier>{feeTier / 10000}%</FeeTier>}
```

avoids showing 0 if its 0